### PR TITLE
Move sdk and openfaas lib out of this repo

### PR DESCRIPTION
**Pull request by s8sg**
The new home for sdk:
> [https://github.com/faasflow/sdk](https://github.com/faasflow/sdk)

The new home for Openfaas flow library
> [https://github.com/faasflow/lib](https://github.com/faasflow/lib)

### CODE REVIEW

File: template/faas-flow/go.mod [hunk 1]
```diff
@@ -1,23 +1,17 @@
 module handler
 
-go 1.14
+go 1.13
```
Comment (pasdam) [inline template/faas-flow/go.mod:3]: Any specific reason for downgrading this?

File: template/faas-flow/.idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```
Comment (pasdam) [inline template/faas-flow/.idea/.gitignore:1]: Probably better not commit the `.idea` folder and add an entry to the `.gitignore` file

File: template/faas-flow/go.mod (hunk 1 above)
Comment (s8sg) [reply template/faas-flow/go.mod:3]: I think its a mistake

Comment (s8sg) [reply template/faas-flow/go.mod]: Fixed

File: template/faas-flow/.idea/.gitignore (hunk 2 above)
Comment (s8sg) [reply template/faas-flow/.idea/.gitignore]: Fixed

**Pull request merged**